src/main/resources/db/changelog/changes/v1.0.0/sql/2021-07-09-02__alter-location-mapping-table.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE location_mapping
+    DROP CONSTRAINT fk_location_mapping_central_server;
+
+ALTER TABLE location_mapping
+    ADD CONSTRAINT fk_location_mapping_central_server FOREIGN KEY (central_server_id)
+        REFERENCES central_server (id)
+        ON DELETE CASCADE;
```

Comment:
It seems you forgot to alter another foreign key in this table, which is `fk_location_mapping_ir_location FOREIGN KEY (ir_location_id) REFERENCES inn_reach_location (id).`

Reply:
Missing constraint added.
According to PostgreSQL system information all FKs have been covered now:

![image](https://user-images.githubusercontent.com/42244720/125427837-289c9c24-ccb7-4e9f-be2d-df21179c8f1c.png)